checkAPIKey.js

```diff
@@ -2,9 +2,20 @@ const Nano = require('nano')
 const nano = Nano(process.env.COUCH_URL)
 const db = nano.db.use(process.env.COUCH_KEYS_DATABASE)
 
+// cache of API keys
+const keyCache = {
```

Comment:
Should we use an LRU cache instead?

If we invalidate a key, it would remain valid as far as the check was concerned until the instance was restarted. Also, if we have enough api keys generated in the future, this could amount to a sizeable memory usage if we don't expire or limit the number of keys in the object